assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -68,22 +59,45 @@ function TemplateDetail() {
   }, [fetchTemplate, templateId]);
 
   return (
-    <PageContainer>
-      <PageColumn>
-        {template && (
-          <FontProvider>
-            <TransformProvider>
-              <CardGallery>
-                {template.pages.map((page) => (
-                  <PreviewPage key={page.id} page={page} />
-                ))}
-              </CardGallery>
-            </TransformProvider>
-          </FontProvider>
-        )}
-      </PageColumn>
-      <PageColumn />
-    </PageContainer>
+    template && (
+      <FontProvider>
+        <TransformProvider>
+          <section />
+          <ContentContainer>
+            <ColumnContainer>
+              <Column>
+                <CardGallery>
+                  {template.pages.map((page) => (
+                    <PreviewPage key={page.id} page={page} />
+                  ))}
+                </CardGallery>
+              </Column>
+              <Column>
+                <DetailContainer>
+                  <Title>{`${template.title} Template`}</Title>
```

Comment:
Can you add in the wordpress i18n web-stories function for all these strings, so this would be 
```
sprintf(_n('%s Template',, 'web-stories'), template.title);
```
... i think?

Reply:
Ooooo, yes, I totally forgot about the i18n function, thanks for the call out, I'll update!